recipes-devtools/codechecker/codechecker_git.bb

```diff
@@ -5,23 +5,59 @@ LIC_FILES_CHKSUM = "file://LICENSE.TXT;md5=2e982d844baa4df1c80de75470e0c5cb"
 
 DEPENDS = "doxygen-native curl-native git-native nodejs-native python3-native"
 
-SRC_URI = " git://github.com/Ericsson/codechecker.git;protocol=https;branch=release-v6.15.2"
+# Codemirror is normaly download by codechecker's build system in codechecker/tools/plist_to_html/Makefile
+# but that doesn't works well with offline build.
+CODEMIRROR_VERSION = "5.30.0"
+CODEMIRROR_BASE_URL = "https://cdnjs.cloudflare.com/ajax/libs/codemirror/${CODEMIRROR_VERSION}"
+
+SRC_URI = " git://github.com/Ericsson/codechecker.git;protocol=https;branch=release-v6.15.2 \
+            ${CODEMIRROR_BASE_URL}/codemirror.min.js;name=codemirror.min.js \
+            ${CODEMIRROR_BASE_URL}/codemirror.min.css;name=codemirror.min.css \
+            https://raw.githubusercontent.com/codemirror/CodeMirror/${CODEMIRROR_VERSION}/LICENSE;name=codemirror.LICENSE;downloadfilename=codemirror.LICENSE \
+            ${CODEMIRROR_BASE_URL}/mode/clike/clike.min.js;name=clike.min.js"
```

Comment:
@samdolt I think the files should include a version number otherwise they will conflict if uploaded to a source mirror.

Reply:
You are right. It's now fixed.